x/ecocredit/core/msg_create_class.go

```diff
@@ -46,6 +46,17 @@ func (m *MsgCreateClass) ValidateBasic() error {
 		}
 	}
 
+	if m.Fee != nil {
```

Comment:
ok, if it's required, and a struct is small (even if we use Coin), then we can add option for `nullable = false`.